src/Microsoft.Data/DataFrameColumn.cs

```diff
@@ -87,9 +87,15 @@ public void SetName(string newName, DataFrame dataFrame = null)
         /// <param name="mapIndices"></param>
         /// <param name="invertMapIndices"></param>
         /// <returns></returns>
-        public virtual DataFrameColumn Clone(DataFrameColumn mapIndices = null, bool invertMapIndices = false, long numberOfNullsToAppend = 0) => throw new NotImplementedException();
+        public virtual DataFrameColumn Clone(DataFrameColumn mapIndices = null, bool invertMapIndices = false, long numberOfNullsToAppend = 0) => CloneImplementation(mapIndices, invertMapIndices, numberOfNullsToAppend);
 
-        public virtual DataFrameColumn Sort(bool ascending = true) => throw new NotImplementedException();
+        protected virtual DataFrameColumn CloneImplementation(DataFrameColumn mapIndices, bool invertMapIndices, long numberOfNullsToAppend) => throw new NotImplementedException();
```

Comment:
This is so `PrimitiveDataFrameColumn` can hide the base class' `Clone` method with `new`. Similar pattern for `FillNulls`, 'Clip` and `Filter`